Automattic/jetpack
# VideoPress: restore pagination section

Fixes https://github.com/Automattic/jetpack/issues/26575

#### Changes proposed in this Pull Request:

* Fetch the initial list of videos on the backend and provide it as the initial state for the client
* Provide the number of pages and the total of videos as the initial state as well
* Avoid to perform the async request adding isFulfilled handler

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






*To test the changes on the initial state:*

* Go to the VideoPress library (`Jetpack > VideoPress`)
* Open the development console on the browser
* Check the content of the following variables:
   * `jetpackVideoPressInitialState.initialState.videos.items` should contain the list of the videos
   * `jetpackVideoPressInitialState.initialState.videos.pagination` should contain the number of pages and the total of videos
* Test with a site with videos
* Confirm pagination is rendered according to the number of videos
* Confirm no async request in the every first rendering
* Confirm pagination and filters work as expected
